### ISSUE TITLE
ClickOutside not working on iOS devices

### ISSUE THREAD

**Issue by docmars** [vue-bulma/click-outside#9]
I haven't done extensive testing on this, but in using the latest version of this package straight from GitHub, rather than npm, the events failed to fire on iOS for some reason. The only device I tried this on was an iPhone 7, though I'm unsure of the Safari version (I don't own an iOS device).

I found a workaround through a similar package for React—[kentor/react-click-outside](https://github.com/kentor/react-click-outside), where their README mentions needing `cursor: pointer` on the `document.body` for iOS to see the event here:
https://github.com/kentor/react-click-outside#not-working-on-ios

After I did this, `vue-click-outside` started working like a charm on a buddy's iPhone.

I'm not sure if this is something worth implementing in the package itself, or if it should be mentioned in the README since there are a number of ways to handle feature detection and adding this workaround.

Thanks!

**Comment by AustinPaquette** [comment]
My first thought is https://github.com/vue-bulma/click-outside/blob/45629f43911abfe9bcba3172e2a3f723a0fa7ce5/index.js#L56

It specifies click, but I'd typically have a touch event recognized as well.

```javascript
const clickHandler = 'ontouchstart' in document.documentElement ? 'touchstart' : 'click';
```

**Comment by brennenf** [comment]
@paquette The above fix worked for me. Thank you.

**Comment by Paquette** [comment]
Glad to help @brennenf totally forgot about this, I haven't made a PR yet!

**Comment by brennenf** [comment]
@Paquette Actually, if you have the time would you mind making a PR? It would be helpful for project install automation to have the patched version on npm. No worries if not!

**Comment by Paquette** [comment]
@brennenf made the PR :)

**Comment by trollepierre** [comment]
So, is that issue closed?

**Comment by littlee** [comment]
😞

**Comment by vasisouv** [comment]
This package looks abandoned. For anyone trying to fix this, try : https://github.com/ndelvalle/v-click-outside, it supports touch events out of the box and was good for my case.

**Comment by pqt** [comment]
Yeah it seems like it, I've had an open PR that's received no feedback or sign of merge whatsoever.

https://github.com/vue-bulma/click-outside/pull/17

I have left the PR open and branch for so long because when I closed it and cleaned up some people emailed me asking where it had disappeared to, was hoping to get this merged sometime before the year 2045 but not looking hopeful at this point 😅

**Comment by pqt** [comment]
Sweet, glad this is finally merged.

**Comment by fundon** [comment]
Thanks.

**Comment by fundon** [comment]
https://github.com/vue-bulma/click-outside/compare/45629f43911abfe9bcba3172e2a3f723a0fa7ce5...v1.1.0 
@pqt  Published v1.1.0
